pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -101,8 +100,15 @@ def __verify_eval_loop_configuration(self, model: LightningModule, stage: str) -
         if has_step and not has_loader:
             rank_zero_warn(f'you defined a {step_name} but have no {loader_name}. Skipping {stage} loop')
 
-    def __verify_predict_loop_configuration(self, model: LightningModule) -> None:
-
+    def __verify_predict_loop_configuration(self, model: 'pl.LightningModule') -> None:
         has_predict_dataloader = is_overridden('predict_dataloader', model)
         if not has_predict_dataloader:
             raise MisconfigurationException('Dataloader not found for `Trainer.predict`')
+
+    def __verify_dp_batch_transfer_support(self, model: 'pl.LightningModule') -> None:
+        """Raise Misconfiguration exception since these hooks are not supported in DP mode"""
+        # TODO: Remove this blocker once batch transfer to device is integrated in Lightning for DP mode.
```

Comment:
is there an issue we can file alongside the TODO to track this?

Reply:
I don't think so.

I just moved the function, cc: @rohitgr7 who added this in #6098

Reply:
yeah, it doesn't work with DP yet. here is the tracking issue you can add if you want: https://github.com/PyTorchLightning/pytorch-lightning/issues/2350